Safewaan/HonkedIn
# Add styling for create/edit profile page

## Trello ticket link

[Developer Task: Implement UI/UX feedback to user profile page](https://trello.com/c/exjqXy6H/128-developer-task-implement-ui-ux-feedback-to-user-profile-page)
[As a user, I want to be able to upload a profile picture for my profile page.](https://trello.com/c/iQXjZAx4/129-as-a-user-i-want-to-be-able-to-upload-a-profile-picture-for-my-profile-page)


## Implementation description
* Rebuilt "My Profile" page with Chakra components
* Added functionality for users to use a URL to an image as a pfp
* Updated styling to match application
* Updated Network Profile page to match styling



## Steps to test
1. Verify that you can update your profile
2. Verify that as a new user (meaning, you have no profile -> either create a new account or delete your profile from the db manually), you can create a profile
3. Verify that you can use a link to an image as a pfp, and that it renders correctly
4. Verify that there are no styling inconsistencies/issues
5. Verify that clicking on another user's profile has the same styling
6. Verify that a "edit profile" button does not appear
